src/styles/card.scss

```diff
@@ -0,0 +1,85 @@
+.catalog__card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  align-items: center;
+  justify-content: space-between;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+  transition: 300ms;
```

Comment:
It's better to specify to what properties you need to apply the transition.

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/08732d91-839b-4c52-b3d1-702228098790)